src/main/java/org/polyfrost/polynametag/mixin/Mixin_RendererLivingEntity_EnableSelfNametag.java

```diff
@@ -7,10 +7,17 @@
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Redirect;
+//#if MC > 1.17.1
+//$$ import com.llamalad7.mixinextras.injector.ModifyReturnValue;
+//$$ import net.minecraft.client.MinecraftClient;
+//$$ import net.minecraft.client.render.entity.LivingEntityRenderer;
+//$$ import net.minecraft.entity.LivingEntity;
+//#endif
 
 @Mixin(RendererLivingEntity.class)
 public class Mixin_RendererLivingEntity_EnableSelfNametag {
 
+    //#if MC < 1.17.1
     @Redirect(method = "canRenderName(Lnet/minecraft/entity/EntityLivingBase;)Z", at = @At(value = "FIELD", target = "Lnet/minecraft/client/renderer/entity/RenderManager;livingPlayer:Lnet/minecraft/entity/Entity;"))
```

Comment:
while we're at it, let's change this Redirect into something else via mixinextras

Reply:
alright its ok then just leave it alone